mcs/class/System/Mono.AppleTls/AppleTlsContext.cs

```diff
@@ -810,13 +799,12 @@ public override unsafe int Read (byte[] buffer, int offset, int count, out bool
 					 * when the first inner Read() returns 0.  MobileAuthenticatedStream.InnerRead() attempts
 					 * to distinguish between a graceful close and abnormal termination of connection.
 					 */
-					wantMore = false;
-					return 0;
+					return (0, false);
 				}
 
 				CheckStatusAndThrow (status, SslStatus.WouldBlock, SslStatus.ClosedGraceful);
-				wantMore = status == SslStatus.WouldBlock;
-				return (int)processed;
+				var wantMore = status == SslStatus.WouldBlock;
+				return ((int)processed, wantMore);
```

Comment:
The native signature is `OSStatus SSLWrite(SSLContextRef context, const void *data, size_t dataLength, size_t *processed)`, so `processed` is `IntPtr` in C#.